openedx_tagging/core/tagging/models/base.py

```diff
@@ -404,6 +404,20 @@ def _find_object_tag_index(tag_ref, object_tags) -> int:
                 -1,
             )
 
+        def _check_current_tag_count() -> None:
+            """
+            Checks if the current count of tags for the object is less than 100
+            """
+            # Exclude self.id to avoid counting the tags that are going to be updated
+            current_count = ObjectTag.objects.filter(object_id=object_id).exclude(taxonomy_id=self.id).count()
+
+            if current_count >= 100:
+                raise ValueError(
+                    _(f"Object ({object_id}) already have 100 or more tags.")
```

Comment:
nit: just a small spelling fix, "already have" -> "already has"

Reply:
Changed here: https://github.com/openedx/openedx-learning/blob/432e4c22eb18a05eeafb5070a14666680f4c1484/openedx_tagging/core/tagging/models/base.py#L458

We are talking about the future state, not the current.